docs/source/endpoints/relations.md

```diff
@@ -0,0 +1,300 @@
+
+Relations can be deleted by relation name, source object, target object, or a combination of these.
+Relations can also be deleted by providing a list of single relations.
+
+If a deleted relation is based on a `RelationChoice` or `RelationList` field, the value of the field is removed or updated accordingly on the source object.
+
+### Delete a list of single relations
+
+by **path**:
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../../src/plone/restapi/tests/http-examples/relations_del_path.req
+```
+
+```{literalinclude} ../../../src/plone/restapi/tests/http-examples/relations_del_path.resp
+:language: http
+```
+
+by **UID**:
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../../src/plone/restapi/tests/http-examples/relations_del_uid.req
+```
+
+```{literalinclude} ../../../src/plone/restapi/tests/http-examples/relations_del_uid.resp
+:language: http
+```
+
+**Failures** are listed in response:
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../../src/plone/restapi/tests/http-examples/relations_del_failure.req
+```
+
+```{literalinclude} ../../../src/plone/restapi/tests/http-examples/relations_del_failure.resp
+:language: http
+```
+
+### Delete relations by relation name
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../../src/plone/restapi/tests/http-examples/relations_del_relationname.req
+```
+
+<!-- Uncomment with https://github.com/plone/plone.api/pull/502 merged
+```{literalinclude} ../../../src/plone/restapi/tests/http-examples/relations_del_relationname.resp
+:language: http
+``` -->
+
+### Delete relations by source
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../../src/plone/restapi/tests/http-examples/relations_del_source.req
+```
+
+```{literalinclude} ../../../src/plone/restapi/tests/http-examples/relations_del_source.resp
+:language: http
+```
+
+Delete relations by source UID or source path.
+
+### Delete relations by target
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../../src/plone/restapi/tests/http-examples/relations_del_target.req
+```
+
+```{literalinclude} ../../../src/plone/restapi/tests/http-examples/relations_del_target.resp
+:language: http
+```
+
+Delete relations by target UID or target path.
+
+### Delete relations by combination of source/target and relation name
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../../src/plone/restapi/tests/http-examples/relations_del_combi.req
+```
+
+```{literalinclude} ../../../src/plone/restapi/tests/http-examples/relations_del_combi.resp
+:language: http
+```
+
+
+## Fix relations
+
+Broken relations can be fixed by releasing and re-indexing them.
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../../src/plone/restapi/tests/http-examples/relations_rebuild.req
+```
+
```

Comment:
Does this request return a response?

Reply:
No, it's a '204 No Content'

Reply:
Incorporated into narrative text to avoid a pedantic example.